bytecode/concrete.py

```diff
@@ -116,23 +116,28 @@ def disassemble(cls, lineno, code, offset):
 
 class ConcreteBytecode(_bytecode.BaseBytecode, list):
 
-    def __init__(self):
+    def __init__(self, instructions=[]):
```

Comment:
Oh, it's a deliberate choice to not accept instructions here. It's because instructions depends on other Bytecode attributes: consts, names, varnames. For example, I prefer to first declare constants and then add a LOAD_CONST using a constant.

If you really want to accept a list of instructions, I suggest to accept also optional consts, names and varnames parameters.